code/game/objects/items/knives.dm

```diff
@@ -190,6 +189,7 @@
 	desc = "A sharpened bone. The bare minimum in survival."
 	embedding = list("pain_mult" = 4, "embed_chance" = 35, "fall_chance" = 10, "armour_block" = 40)
 	force = 15
+	armour_penetration = 50
```

Comment:
why are you making all the bone options stronger than other options?  Spears and arrows are piercing weapons, after all. These, not so much. Being bone doesn't magically make something pierce armor

Reply:
It sure seems to for spears and arrows, You've pointed out the bone dagger which very much is a piercing weapon as all daggers are.